src/GT.Core/Services/Impl/GTListingService.cs

```diff
@@ -89,29 +89,44 @@ public class GTListingService : IGTListingService
 				{
 					if (!await _companyService.ExistsByNameAsync(listingDTO.Employer))
 					{
-						await _companyService.AddAsync(new CompanyDTO() { Name = listingDTO.Employer });
+						await _companyService
+							.AddAsync(new CompanyDTO() { Name = listingDTO.Employer });
 					}
-					entity.Employer = await _companyRepository.GetAll().Where(e => e.Name == listingDTO.Employer).FirstOrDefaultAsync();
+					var emp = await _companyRepository
```

Comment:
Var name should be employer as we have been rather explicit with variable names.